1.cpp

```diff
@@ -2,6 +2,7 @@
 #include <bits/stdc++.h>
 using namespace std ;
 // abcdef
+//Newchange
```

Comment:
Dont add comments like this. Keep in mind next time.

Reply:
Will keep in mind next time.